core/src/main/java/io/questdb/std/str/Utf8s.java

```diff
@@ -153,15 +153,24 @@ public static boolean endsWithLowerCaseAscii(@NotNull Utf8Sequence seq, @NotNull
         return !(seqSize == 0 || seqSize < size) && equalsAsciiLowerCase(asciiEnds, seq, seqSize - size, seqSize);
     }
 
+    /**
+     * Return true if the sequences contain equal strings. Co-exists with
+     * {@link #equals(Utf8Sequence, Utf8Sequence)} merely to avoid megamorphism
+     * in {@link Utf8StringIntHashMap} and {@link Utf8StringObjHashMap}. Also, unlike
```

Comment:
this is not a valid javadoc and will generate warnings

Reply:
Improved in 31342e37c3cc18705b39334e27ef4ada5e8a9d67